drivers/gpio/Kconfig

```diff
@@ -178,4 +178,6 @@ source "drivers/gpio/Kconfig.xlnx_ps"
 source "drivers/gpio/Kconfig.xmc4xxx"
 # zephyr-keep-sorted-stop
 
+source "drivers/gpio/Kconfig.max22190"
```

Comment:
Move this up into the `zephyr-keep-sorted` section

Reply:
Done